tests/lib/cisco_test.py

```diff
@@ -657,6 +662,47 @@ def testObjectGroup(self):
                                              mock.call('SOME_HOST')])
     self.naming.GetServiceByProto.assert_called_once_with('HTTP', 'tcp')
 
+  def testObjectGroupInet6(self):
+    ip_grp = ['object-group network ipv6 SOME_HOST']
+    ip_grp.append(' 2001::3/128')
+    ip_grp.append('exit')
+    port_grp1 = ['object-group port 80-80']
+    port_grp1.append(' eq 80')
+    port_grp1.append('exit')
+    port_grp2 = ['object-group port 1024-65535']
+    port_grp2.append(' range 1024 65535')
+    port_grp2.append('exit')
+
+    self.naming.GetNetAddr.return_value = [
+        nacaddr.IP('2001::3/128', token='SOME_HOST')]
```

Comment:
Could you have this also return an IPv4 address, and then we can add a negative test (assertNotIn) for that IPv4 address to ensure IPv4 addresses are not rendered in this object group.

Reply:
I'm happy to update the tests, but I'm not exactly sure what you want me to test here. If `GetNetAddr` returns a list including an IPv4 address, then an IPv4 object-group is still created in the policy output. However, that isn't something that was updated in this PR.

I may be misunderstanding, because it seems like all network object groups are created, regardless of address family. See [here](https://github.com/google/capirca/blob/f844501277d057c12a79148839f595762763ebdf/capirca/lib/cisco.py#L250-L254).